src/index.js

```diff
@@ -129,13 +130,14 @@ export function filterCards(cards, options) {
 }
 
 export function deckFromCards(cards, images, options) {
+	// get path to highlight.js cdn directory
+	let cssPath = path.dirname(resolve.sync('@highlightjs/cdn-assets/package.json'));
+	// construct path to highlight.js stylesheet
+	cssPath = path.resolve(cssPath, `styles/${options.codeStyle}.min.css`);
 	// load syntax-highlighting css
-	let css;
-	try{ // why can't I just import CSS like es6 modules ?!
-		css = fs.readFileSync(path.resolve(__dirname, `../../@highlightjs/cdn-assets/styles/${options.codeStyle}.min.css`),'utf8');
-	}catch(err){
-		css = fs.readFileSync(path.resolve(__dirname, `../node_modules/@highlightjs/cdn-assets/styles/${options.codeStyle}.min.css`),'utf8');
-	}
+	let css = fs.readFileSync(cssPath,'utf8');
+	// remove comment from css, because that breaks things for some reason
+	css = css.replace(/\/\*[\s\S]*\*\//gm, '');
```

Comment:
Can we remove this duplicate line? Or does this have any use?

```suggestion
```

Reply:
Oh no, I was just very tired yesterday 😅